{
	"alias": "FEDORA-2024-afba3b5902",
	"autokarma": true,
	"autotime": true,
	"bugs": [
		{
			"bug_id": 2307370,
			"feedback": [
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3720960,
						"karma": 1,
						"timestamp": "2024-09-15 19:03:01",
						"update_id": 646827,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/f80c96a3994c2479fc4815a146079bf93ade230fe3f3993213d2b8222cafc73d?s=24&d=retro",
							"email": "fzatlouk@redhat.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "qa-tools-sig"
								},
								{
									"name": "provenpackager"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "qa-admin"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "sysadmin-qa"
								},
								{
									"name": "sysadmin"
								},
								{
									"name": "ambassadors"
								},
								{
									"name": "advocates"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "qa"
								},
								{
									"name": "ipausers"
								}
							],
							"id": 2240,
							"name": "frantisekz",
							"openid": "frantisekz.id.fedoraproject.org"
						},
						"user_id": 2240
					},
					"comment_id": 3720960
				},
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3696993,
						"karma": 1,
						"text": "Works",
						"timestamp": "2024-08-29 19:13:53",
						"update_id": 643959,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/c12c2b537990fc7c6e095d9a2e4c4ffe07cb228788849f65861dcb112444ca4c?s=24&d=retro",
							"email": "derekenz@gmail.com",
							"groups": [
								{
									"name": "ipausers"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 8341,
							"name": "derekenz",
							"openid": "derekenz.id.fedoraproject.org"
						},
						"user_id": 8341
					},
					"comment_id": 3696993
				},
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3698017,
						"karma": 1,
						"timestamp": "2024-08-30 07:05:19",
						"update_id": 643951,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/2b94c96c63ae9341f1af7e212918609b3c67be8deae4cfb2de732806dbde6f1d?s=24&d=retro",
							"email": "vtrefny@redhat.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "gitanaconda"
								},
								{
									"name": "storage_apis"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 892,
							"name": "vtrefny",
							"openid": "vtrefny.id.fedoraproject.org"
						},
						"user_id": 892
					},
					"comment_id": 3698017
				},
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3698414,
						"karma": 1,
						"text": "Works",
						"timestamp": "2024-08-30 19:07:39",
						"update_id": 643956,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/c12c2b537990fc7c6e095d9a2e4c4ffe07cb228788849f65861dcb112444ca4c?s=24&d=retro",
							"email": "derekenz@gmail.com",
							"groups": [
								{
									"name": "ipausers"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 8341,
							"name": "derekenz",
							"openid": "derekenz.id.fedoraproject.org"
						},
						"user_id": 8341
					},
					"comment_id": 3698414
				},
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3701408,
						"text": "Is this the correct commit fix? as I understand the commit from this pr https://github.com/python/cpython/pull/123161. But the real fix for the 3.9 branch is here https://github.com/python/cpython/pull/123432",
						"timestamp": "2024-09-02 12:20:14",
						"update_id": 643933,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/ee174ccfe37d8be88350b84ed403afbafece548af3d908c209b47b22fb3e1cef?s=24&d=retro",
							"email": "ilia.polyvyanyy@red-soft.ru",
							"groups": [
								{
									"name": "ipausers"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 8376,
							"name": "polyvyanyy",
							"openid": "polyvyanyy.id.fedoraproject.org"
						},
						"user_id": 8376
					},
					"comment_id": 3701408
				},
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3698025,
						"karma": 1,
						"timestamp": "2024-08-30 07:08:11",
						"update_id": 643961,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/2b94c96c63ae9341f1af7e212918609b3c67be8deae4cfb2de732806dbde6f1d?s=24&d=retro",
							"email": "vtrefny@redhat.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "gitanaconda"
								},
								{
									"name": "storage_apis"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 892,
							"name": "vtrefny",
							"openid": "vtrefny.id.fedoraproject.org"
						},
						"user_id": 892
					},
					"comment_id": 3698025
				},
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3706903,
						"karma": -1,
						"text": "https://github.com/python/cpython/pull/123432 now merged",
						"timestamp": "2024-09-05 12:34:38",
						"update_id": 643933,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/ee174ccfe37d8be88350b84ed403afbafece548af3d908c209b47b22fb3e1cef?s=24&d=retro",
							"email": "ilia.polyvyanyy@red-soft.ru",
							"groups": [
								{
									"name": "ipausers"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 8376,
							"name": "polyvyanyy",
							"openid": "polyvyanyy.id.fedoraproject.org"
						},
						"user_id": 8376
					},
					"comment_id": 3706903
				},
				{
					"bug_id": 2307370,
					"comment": {
						"id": 3713552,
						"karma": 1,
						"timestamp": "2024-09-10 09:46:10",
						"update_id": 646827,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/2b94c96c63ae9341f1af7e212918609b3c67be8deae4cfb2de732806dbde6f1d?s=24&d=retro",
							"email": "vtrefny@redhat.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "gitanaconda"
								},
								{
									"name": "storage_apis"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 892,
							"name": "vtrefny",
							"openid": "vtrefny.id.fedoraproject.org"
						},
						"user_id": 892
					},
					"comment_id": 3713552
				}
			],
			"parent": true,
			"security": true,
			"title": "CVE-2024-8088 python: cpython: Iterating over a malicious ZIP file may lead to Denial of Service",
			"bugzilla": {
				"bug_id": "2307370",
				"alias": "CVE-2024-8088",
				"creation_ts": "2024-08-22 19:20:34 +0000",
				"short_desc": "CVE-2024-8088 python: cpython: Iterating over a malicious ZIP file may lead to Denial of Service",
				"delta_ts": "2024-09-24 00:48:47 +0000",
				"bug_status": "NEW",
				"keywords": "Security",
				"priority": "medium",
				"bug_severity": "medium",
				"depends_on": [
					"2307465",
					"2307457",
					"2307458",
					"2307459",
					"2307460",
					"2307461",
					"2307462",
					"2307463",
					"2307464",
					"2307466"
				],
				"external_bugs": {
					"text": "RHSA-2024:6962",
					"name": "Red Hat Product Errata"
				},
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "18159903",
						"comment_count": "0",
						"who": {
							"text": "bzimport",
							"name": "OSIDB Bzimport"
						},
						"bug_when": "2024-08-22 19:20:34 +0000",
						"thetext": "There is a HIGH severity vulnerability affecting the CPython \"zipfile\"\nmodule.\n\n\n\n\n\nWhen iterating over names of entries in a zip archive (for example, methods\nof \"zipfile.ZipFile\" like \"namelist()\", \"iterdir()\", \"extractall()\", etc)\nthe process can be put into an infinite loop with a maliciously crafted\nzip archive. This defect applies when reading only metadata or extracting\nthe contents of the zip archive. Programs that are not handling\nuser-controlled zip archives are not affected."
					},
					{
						"isprivate": "0",
						"commentid": "18165593",
						"comment_count": "1",
						"who": {
							"text": "errata-xmlrpc",
							"name": "errata-xmlrpc"
						},
						"bug_when": "2024-08-28 18:51:55 +0000",
						"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Enterprise Linux 8\n\nVia RHSA-2024:5962 https://access.redhat.com/errata/RHSA-2024:5962"
					},
					{
						"isprivate": "0",
						"commentid": "18184757",
						"comment_count": "2",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-12 01:27:58 +0000",
						"thetext": "FEDORA-2024-e887a10dee (python3.13-3.13.0~rc2-1.fc40) has been pushed to the Fedora 40 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18190050",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-17 02:07:35 +0000",
						"thetext": "FEDORA-2024-f2fc325c40 (python3.13-3.13.0~rc2-1.fc39) has been pushed to the Fedora 39 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18199601",
						"comment_count": "4",
						"who": {
							"text": "errata-xmlrpc",
							"name": "errata-xmlrpc"
						},
						"bug_when": "2024-09-24 00:48:32 +0000",
						"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Enterprise Linux 8\n\nVia RHSA-2024:6961 https://access.redhat.com/errata/RHSA-2024:6961"
					},
					{
						"isprivate": "0",
						"commentid": "18199602",
						"comment_count": "5",
						"who": {
							"text": "errata-xmlrpc",
							"name": "errata-xmlrpc"
						},
						"bug_when": "2024-09-24 00:48:45 +0000",
						"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Enterprise Linux 8\n\nVia RHSA-2024:6962 https://access.redhat.com/errata/RHSA-2024:6962"
					}
				]
			}
		},
		{
			"bug_id": 2307461,
			"feedback": [
				{
					"bug_id": 2307461,
					"comment": {
						"id": 3698017,
						"karma": 1,
						"timestamp": "2024-08-30 07:05:19",
						"update_id": 643951,
						"user": {
							"avatar": "https://seccdn.libravatar.org/avatar/2b94c96c63ae9341f1af7e212918609b3c67be8deae4cfb2de732806dbde6f1d?s=24&d=retro",
							"email": "vtrefny@redhat.com",
							"groups": [
								{
									"name": "packager"
								},
								{
									"name": "ipausers"
								},
								{
									"name": "fedora-contributor"
								},
								{
									"name": "signed_fpca"
								},
								{
									"name": "fedorabugs"
								},
								{
									"name": "gitanaconda"
								},
								{
									"name": "storage_apis"
								},
								{
									"name": "trust admins"
								}
							],
							"id": 892,
							"name": "vtrefny",
							"openid": "vtrefny.id.fedoraproject.org"
						},
						"user_id": 892
					},
					"comment_id": 3698017
				}
			],
			"security": true,
			"title": "CVE-2024-8088 python3.12: From NVD collector [fedora-all]",
			"bugzilla": {
				"bug_id": "2307461",
				"creation_ts": "2024-08-22 22:59:30 +0000",
				"short_desc": "CVE-2024-8088 python3.12: From NVD collector [fedora-all]",
				"delta_ts": "2024-09-21 01:27:28 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "Security, SecurityTracking",
				"priority": "medium",
				"bug_severity": "medium",
				"blocked": [
					{
						"bug_id": "2307370",
						"alias": "CVE-2024-8088",
						"creation_ts": "2024-08-22 19:20:34 +0000",
						"short_desc": "CVE-2024-8088 python: cpython: Iterating over a malicious ZIP file may lead to Denial of Service",
						"delta_ts": "2024-09-24 00:48:47 +0000",
						"bug_status": "NEW",
						"keywords": "Security",
						"priority": "medium",
						"bug_severity": "medium",
						"depends_on": [
							"2307465",
							"2307457",
							"2307458",
							"2307459",
							"2307460",
							"2307461",
							"2307462",
							"2307463",
							"2307464",
							"2307466"
						],
						"external_bugs": {
							"text": "RHSA-2024:6962",
							"name": "Red Hat Product Errata"
						},
						"long_desc": [
							{
								"isprivate": "0",
								"commentid": "18159903",
								"comment_count": "0",
								"who": {
									"text": "bzimport",
									"name": "OSIDB Bzimport"
								},
								"bug_when": "2024-08-22 19:20:34 +0000",
								"thetext": "There is a HIGH severity vulnerability affecting the CPython \"zipfile\"\nmodule.\n\n\n\n\n\nWhen iterating over names of entries in a zip archive (for example, methods\nof \"zipfile.ZipFile\" like \"namelist()\", \"iterdir()\", \"extractall()\", etc)\nthe process can be put into an infinite loop with a maliciously crafted\nzip archive. This defect applies when reading only metadata or extracting\nthe contents of the zip archive. Programs that are not handling\nuser-controlled zip archives are not affected."
							},
							{
								"isprivate": "0",
								"commentid": "18165593",
								"comment_count": "1",
								"who": {
									"text": "errata-xmlrpc",
									"name": "errata-xmlrpc"
								},
								"bug_when": "2024-08-28 18:51:55 +0000",
								"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Enterprise Linux 8\n\nVia RHSA-2024:5962 https://access.redhat.com/errata/RHSA-2024:5962"
							},
							{
								"isprivate": "0",
								"commentid": "18184757",
								"comment_count": "2",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2024-09-12 01:27:58 +0000",
								"thetext": "FEDORA-2024-e887a10dee (python3.13-3.13.0~rc2-1.fc40) has been pushed to the Fedora 40 stable repository.\nIf problem still persists, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "18190050",
								"comment_count": "3",
								"who": {
									"text": "updates",
									"name": "Fedora Update System"
								},
								"bug_when": "2024-09-17 02:07:35 +0000",
								"thetext": "FEDORA-2024-f2fc325c40 (python3.13-3.13.0~rc2-1.fc39) has been pushed to the Fedora 39 stable repository.\nIf problem still persists, please make note of it in this bug report."
							},
							{
								"isprivate": "0",
								"commentid": "18199601",
								"comment_count": "4",
								"who": {
									"text": "errata-xmlrpc",
									"name": "errata-xmlrpc"
								},
								"bug_when": "2024-09-24 00:48:32 +0000",
								"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Enterprise Linux 8\n\nVia RHSA-2024:6961 https://access.redhat.com/errata/RHSA-2024:6961"
							},
							{
								"isprivate": "0",
								"commentid": "18199602",
								"comment_count": "5",
								"who": {
									"text": "errata-xmlrpc",
									"name": "errata-xmlrpc"
								},
								"bug_when": "2024-09-24 00:48:45 +0000",
								"thetext": "This issue has been addressed in the following products:\n\n  Red Hat Enterprise Linux 8\n\nVia RHSA-2024:6962 https://access.redhat.com/errata/RHSA-2024:6962"
							}
						]
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "18160110",
						"comment_count": "0",
						"who": {
							"text": "mbenatto",
							"name": "Marco Benatto"
						},
						"bug_when": "2024-08-22 22:59:30 +0000",
						"thetext": "More information about this security flaw is available in the following bug:\n\nhttps://bugzilla.redhat.com/show_bug.cgi?id=2307370\n\nDisclaimer: Community trackers are created by Red Hat Product Security team on a best effort basis. Package maintainers are required to ascertain if the flaw indeed affects their package, before starting the update process."
					},
					{
						"isprivate": "0",
						"commentid": "18165058",
						"comment_count": "1",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-28 11:09:57 +0000",
						"thetext": "FEDORA-2024-0bcad53964 (python3.12-3.12.5-2.fc42) has been submitted as an update to Fedora 42.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-0bcad53964"
					},
					{
						"isprivate": "0",
						"commentid": "18165064",
						"comment_count": "2",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-28 11:14:03 +0000",
						"thetext": "FEDORA-2024-0bcad53964 (python3.12-3.12.5-2.fc42) has been pushed to the Fedora 42 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18165136",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-28 12:17:22 +0000",
						"thetext": "FEDORA-2024-e537311f91 (python3.12-3.12.5-2.fc39) has been submitted as an update to Fedora 39.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-e537311f91"
					},
					{
						"isprivate": "0",
						"commentid": "18165137",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-28 12:17:24 +0000",
						"thetext": "FEDORA-2024-afba3b5902 (python3.12-3.12.5-2.fc40) has been submitted as an update to Fedora 40.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-afba3b5902"
					},
					{
						"isprivate": "0",
						"commentid": "18165138",
						"comment_count": "5",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-28 12:17:25 +0000",
						"thetext": "FEDORA-2024-b673d8c70f (python3.12-3.12.5-2.fc41) has been submitted as an update to Fedora 41.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-b673d8c70f"
					},
					{
						"isprivate": "0",
						"commentid": "18165844",
						"comment_count": "6",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-29 01:59:17 +0000",
						"thetext": "FEDORA-2024-b673d8c70f has been pushed to the Fedora 41 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-b673d8c70f`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-b673d8c70f\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18165883",
						"comment_count": "7",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-29 02:38:37 +0000",
						"thetext": "FEDORA-2024-e537311f91 has been pushed to the Fedora 39 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-e537311f91`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-e537311f91\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18165902",
						"comment_count": "8",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-08-29 02:41:18 +0000",
						"thetext": "FEDORA-2024-afba3b5902 has been pushed to the Fedora 40 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-afba3b5902`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-afba3b5902\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18171906",
						"comment_count": "9",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-03 05:50:07 +0000",
						"thetext": "FEDORA-2024-afba3b5902 (python3.12-3.12.5-2.fc40) has been pushed to the Fedora 40 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18182234",
						"comment_count": "10",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-10 16:10:28 +0000",
						"thetext": "FEDORA-2024-6def755315 (python3.12-3.12.6-1.fc41) has been submitted as an update to Fedora 41.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-6def755315"
					},
					{
						"isprivate": "0",
						"commentid": "18182235",
						"comment_count": "11",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-10 16:10:29 +0000",
						"thetext": "FEDORA-2024-1d0cb3b43f (python3.12-3.12.6-1.fc40) has been submitted as an update to Fedora 40.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-1d0cb3b43f"
					},
					{
						"isprivate": "0",
						"commentid": "18182236",
						"comment_count": "12",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-10 16:10:30 +0000",
						"thetext": "FEDORA-2024-e453a209e9 (python3.12-3.12.6-1.fc39) has been submitted as an update to Fedora 39.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-e453a209e9"
					},
					{
						"isprivate": "0",
						"commentid": "18182920",
						"comment_count": "13",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-11 02:29:01 +0000",
						"thetext": "FEDORA-2024-6def755315 has been pushed to the Fedora 41 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-6def755315`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-6def755315\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18182951",
						"comment_count": "14",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-11 02:54:50 +0000",
						"thetext": "FEDORA-2024-e453a209e9 has been pushed to the Fedora 39 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-e453a209e9`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-e453a209e9\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18182974",
						"comment_count": "15",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-11 02:57:43 +0000",
						"thetext": "FEDORA-2024-1d0cb3b43f has been pushed to the Fedora 40 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-1d0cb3b43f`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-1d0cb3b43f\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18187288",
						"comment_count": "16",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-13 20:45:18 +0000",
						"thetext": "FEDORA-2024-b673d8c70f (python3.12-3.12.5-2.fc41) has been pushed to the Fedora 41 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18187732",
						"comment_count": "17",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-14 01:58:34 +0000",
						"thetext": "FEDORA-2024-1d0cb3b43f (python3.12-3.12.6-1.fc40 and python3-docs-3.12.6-1.fc40) has been pushed to the Fedora 40 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18194141",
						"comment_count": "18",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-19 00:17:24 +0000",
						"thetext": "FEDORA-2024-6def755315 (python3.12-3.12.6-1.fc41) has been pushed to the Fedora 41 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18196892",
						"comment_count": "19",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-09-21 01:27:28 +0000",
						"thetext": "FEDORA-2024-e453a209e9 (python3.12-3.12.6-1.fc39 and python3-docs-3.12.6-1.fc39) has been pushed to the Fedora 39 stable repository.\nIf problem still persists, please make note of it in this bug report."
					}
				]
			}
		}
	],
	"builds": [
		{
			"epoch": 0,
			"nvr": "python3.12-3.12.5-2.fc40",
			"release_id": 76,
			"signed": true,
			"type": "rpm",
			"package": {
				"aarch64": [
					{
						"name": "python3",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3-devel",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3-tkinter",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3-debug",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3-idle",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3.12-debugsource",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3.12-debuginfo",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3-libs",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					},
					{
						"name": "python3-test",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "aarch64"
					}
				],
				"i686": [
					{
						"name": "python3.12-debugsource",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3.12-debuginfo",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3-libs",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3-test",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3-debug",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3-idle",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3-tkinter",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3-devel",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					},
					{
						"name": "python3",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "i686"
					}
				],
				"noarch": [
					{
						"name": "python-unversioned-command",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "noarch"
					}
				],
				"ppc64le": [
					{
						"name": "python3-tkinter",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3-debug",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3.12-debugsource",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3.12-debuginfo",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3-test",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3-libs",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3-devel",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3-idle",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					},
					{
						"name": "python3",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "ppc64le"
					}
				],
				"s390x": [
					{
						"name": "python3-test",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3-libs",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3.12-debuginfo",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3.12-debugsource",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3-debug",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3-idle",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3-tkinter",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3-devel",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					},
					{
						"name": "python3",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "s390x"
					}
				],
				"src": [
					{
						"name": "python3.12",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "src"
					}
				],
				"x86_64": [
					{
						"name": "python3",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3-devel",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3-tkinter",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3-idle",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3.12-debugsource",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3-debug",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3.12-debuginfo",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3-libs",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					},
					{
						"name": "python3-test",
						"version": "3.12.5",
						"release": "2.fc40",
						"arch": "x86_64"
					}
				]
			}
		}
	],
	"close_bugs": true,
	"comments": [
		{
			"id": 3695675,
			"text": "This update has been submitted for testing by churchyard. ",
			"timestamp": "2024-08-28 12:17:20",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3695677,
			"text": "This update's test gating status has been changed to 'waiting'.",
			"timestamp": "2024-08-28 12:17:21",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3695765,
			"text": "This update's test gating status has been changed to 'passed'.",
			"timestamp": "2024-08-28 14:10:20",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3696437,
			"text": "This update has been pushed to testing.",
			"timestamp": "2024-08-29 02:41:46",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3696531,
			"karma": 1,
			"text": "Works.",
			"timestamp": "2024-08-29 03:00:21",
			"update_id": 643951,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/1a8afd83231ef5ba905344806ba95b39f2abd77b399b0ee217ec3e6fa30a4825?s=24&d=retro",
				"email": "bojan@rexursive.com",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					}
				],
				"id": 198,
				"name": "bojan",
				"openid": "bojan.id.fedoraproject.org"
			},
			"user_id": 198
		},
		{
			"id": 3696988,
			"karma": 1,
			"text": "Works",
			"timestamp": "2024-08-29 19:12:11",
			"update_id": 643951,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/c12c2b537990fc7c6e095d9a2e4c4ffe07cb228788849f65861dcb112444ca4c?s=24&d=retro",
				"email": "derekenz@gmail.com",
				"groups": [
					{
						"name": "ipausers"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "trust admins"
					}
				],
				"id": 8341,
				"name": "derekenz",
				"openid": "derekenz.id.fedoraproject.org"
			},
			"user_id": 8341
		},
		{
			"id": 3697033,
			"karma": 1,
			"timestamp": "2024-08-29 20:09:02",
			"update_id": 643951,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/5545678c86b7a87ee5d907a87573978f63c78b2d498edda166dac72f81003f73?s=24&d=retro",
				"email": "geraldo.simiao.kutz@gmail.com",
				"groups": [
					{
						"name": "qa"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "ambassadors"
					},
					{
						"name": "fedora-br"
					},
					{
						"name": "advocates"
					},
					{
						"name": "respins-sig"
					}
				],
				"id": 5881,
				"name": "geraldosimiao",
				"openid": "geraldosimiao.id.fedoraproject.org"
			},
			"user_id": 5881
		},
		{
			"id": 3697122,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697123,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697124,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697125,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697126,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697127,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697128,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697129,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697130,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697131,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697132,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697133,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697134,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697135,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697136,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697137,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697138,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697139,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697140,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3697141,
			"text": "This update can be pushed to stable now if the maintainer wishes",
			"timestamp": "2024-08-29 20:19:17",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"bug_feedback": [
				{
					"bug": {
						"bug_id": 2307370,
						"parent": true,
						"security": true,
						"title": "CVE-2024-8088 python: cpython: Iterating over a malicious ZIP file may lead to Denial of Service"
					},
					"bug_id": 2307370,
					"comment_id": 3698017
				},
				{
					"bug": {
						"bug_id": 2307461,
						"security": true,
						"title": "CVE-2024-8088 python3.12: From NVD collector [fedora-all]"
					},
					"bug_id": 2307461,
					"comment_id": 3698017
				}
			],
			"id": 3698017,
			"karma": 1,
			"timestamp": "2024-08-30 07:05:19",
			"update_id": 643951,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/2b94c96c63ae9341f1af7e212918609b3c67be8deae4cfb2de732806dbde6f1d?s=24&d=retro",
				"email": "vtrefny@redhat.com",
				"groups": [
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "gitanaconda"
					},
					{
						"name": "storage_apis"
					},
					{
						"name": "trust admins"
					}
				],
				"id": 892,
				"name": "vtrefny",
				"openid": "vtrefny.id.fedoraproject.org"
			},
			"user_id": 892
		},
		{
			"id": 3700956,
			"karma": 1,
			"text": "no issues in regular Workstation usage",
			"timestamp": "2024-09-02 06:24:52",
			"update_id": 643951,
			"user": {
				"avatar": "https://seccdn.libravatar.org/avatar/dcc9b804c7f4931d90ad5f9b8b2518e79df4743a418f53589057dc035bddcd0d?s=24&d=retro",
				"email": "kparal@redhat.com",
				"groups": [
					{
						"name": "qa-tools-sig"
					},
					{
						"name": "proventesters"
					},
					{
						"name": "qa"
					},
					{
						"name": "packager"
					},
					{
						"name": "ipausers"
					},
					{
						"name": "fedora-contributor"
					},
					{
						"name": "signed_fpca"
					},
					{
						"name": "fedorabugs"
					},
					{
						"name": "qa-admin"
					}
				],
				"id": 411,
				"name": "kparal",
				"openid": "kparal.id.fedoraproject.org"
			},
			"user_id": 411
		},
		{
			"id": 3700957,
			"text": "This update has been submitted for stable by bodhi. ",
			"timestamp": "2024-09-02 06:24:52",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3702748,
			"text": "This update has been pushed to stable.",
			"timestamp": "2024-09-03 05:50:28",
			"update_id": 643951,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		}
	],
	"content_type": "rpm",
	"critpath": true,
	"critpath_groups": "core critical-path-anaconda critical-path-apps critical-path-base critical-path-build critical-path-compose critical-path-deepin-desktop critical-path-gnome critical-path-kde critical-path-lxde critical-path-lxqt critical-path-server critical-path-standard critical-path-xfce",
	"date_approved": "2024-09-02 06:24:52",
	"date_pushed": "2024-09-03 05:49:14",
	"date_stable": "2024-09-03 05:49:14",
	"date_submitted": "2024-08-28 12:17:20",
	"date_testing": "2024-08-29 02:40:50",
	"karma": 5,
	"meets_testing_requirements": true,
	"notes": "Security fix for CVE-2024-8088",
	"pushed": true,
	"release": {
		"branch": "f40",
		"candidate_tag": "f40-updates-candidate",
		"composed_by_bodhi": true,
		"create_automatic_updates": false,
		"dist_tag": "f40",
		"eol": "2025-05-13",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 40",
		"mail_template": "fedora_errata_template",
		"name": "F40",
		"override_tag": "f40-override",
		"package_manager": "dnf",
		"pending_signing_tag": "f40-signing-pending",
		"pending_stable_tag": "f40-updates-pending",
		"pending_testing_tag": "f40-updates-testing-pending",
		"stable_tag": "f40-updates",
		"state": "current",
		"testing_repository": "updates-testing",
		"testing_tag": "f40-updates-testing",
		"version": "40"
	},
	"require_bugs": true,
	"require_testcases": true,
	"severity": "medium",
	"stable_days": 14,
	"stable_karma": 5,
	"status": "stable",
	"suggest": "unspecified",
	"test_gating_status": "passed",
	"title": "python3.12-3.12.5-2.fc40",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2024-afba3b5902",
	"unstable_karma": -2,
	"updateid": "FEDORA-2024-afba3b5902",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/132a6c64fa9fbe9311973f5c5c6a70942b256d23d62be638718fe091076731cf?s=24&d=retro",
		"email": "mhroncok@redhat.com",
		"groups": [
			{
				"name": "provenpackager"
			},
			{
				"name": "packager"
			},
			{
				"name": "python-sig"
			},
			{
				"name": "signed_fpca"
			},
			{
				"name": "3d-printing-sig"
			},
			{
				"name": "python-packagers-sig"
			},
			{
				"name": "fedora-contributor"
			},
			{
				"name": "sysadmin-badges"
			},
			{
				"name": "ipausers"
			},
			{
				"name": "sysadmin"
			},
			{
				"name": "lua-packagers-sig"
			},
			{
				"name": "fedorabugs"
			},
			{
				"name": "cvsadmin"
			},
			{
				"name": "ambassadors"
			},
			{
				"name": "packaging-committee"
			},
			{
				"name": "advocates"
			},
			{
				"name": "trust admins"
			}
		],
		"id": 533,
		"name": "churchyard",
		"openid": "churchyard.id.fedoraproject.org"
	},
	"version_hash": "2dbc7f1a9941f2b28661c0b3c16470c5053c8065"
}
